scope & closures/ch5.md

```diff
@@ -23,7 +23,7 @@ O momento de esclarecimento deve ser: **oh, clausuras já estão surgindo por to
 
 OK, chega de exageros e referências sem-vergonhas de filmes.
 
-Here's a down-n-dirty definition of what you need to know to understand and recognize closures:
+Aqui está uma definição suja do que você precisa saber para entender e reconhecer clousure:
```

Comment:
Fiquei na dúvida com relação ao _`down-n-dirty`_. Até consegui achar algumas definições, mas não sei qual seria a mais adequada. O grande problema é que eu não tenho certeza o quanto esse adjetivo tem a contribuir para a compreensão do texto. Mesmo lendo a definição dada por ele após este trecho, continuei na dúvida. Só não aprovei ainda para saber o que vc acha disso.
http://idioms.thefreedictionary.com/down-and-dirty
http://www.yourdictionary.com/down-and-dirty
http://www.urbandictionary.com/define.php?term=the%20down%20and%20dirty
http://www.urbandictionary.com/define.php?term=getting%20down%20and%20dirty